### DISCUSSION TITLE
CSS Module files not working on custom server, but they work just fine when deployed to Vercel.

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#17802] [Help] [answered]
This is somewhat baffling. The project that's being worked on doesn't use `@zeit/next-css`, just the in-built whatever Next provides. It's great, provides great optimizations out of the box and it's good enough. I deploy it successfully to Vercel without a hitch, yet, when I'm trying to deploy on a custom node server, I'm met with this error:

```
> project@0.1.0 build /var/www/vhosts/site.com/nextjs
> next build

info  - Creating an optimized production build...
Failed to compile.

ModuleNotFoundError: Module not found: Error: Can't resolve './IconWithText.module.css' in '/var/www/vhosts/site.com/nextjs/components/Misc'


> Build error occurred
Error: > Build failed because of webpack errors
    at build (/var/www/vhosts/site.com/nextjs/node_modules/next/dist/build/index.js:15:918)
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! project@0.1.0 build: `next build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the project@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```
What's going on here? Where is the disconnect?

**Comment by jamesmosier** [comment, marked as answer]
If you rename the file to something like `icon-with-text.module.css` and import it, does it work?

**Reply by ghost** [reply]
It does. And that's the weirdest thing. I made sure that 100%, no foul-play, the file/path/capitalization is the same and, again, remember, it works on localhost (both Linux & Windows).

**Reply by jamesmosier** [reply]
If you commit a file in Git named `iconWithText.module.css` (lowercase `i` at the beginning) and then change the name to be `IconWithText.module.css` later on and commit that, Git does not recognize the capitalization change. Therefore the easiest way to resolve this would be to delete `IconWithText.module.css` completely, commit it with the file deleted, then recreate the file named properly and commit that.
